src/meta_ext.rs

```diff
@@ -234,14 +233,7 @@ impl<'a> Pallet<'a> for PalletMeta {
     }
 
     fn get_calls(&self) -> Option<&PalletCallMetadata<PortableForm>> {
-        self.calls.iter().find(|_call| {
-            true
-        })
-    }
-
-    fn ty_id(&self) -> u32 {
-        dbg!(self);
-        0u32
+        self.calls.iter().find(|_| true)
```

Comment:
This `.calls.iter().find(|_| true)` still looks pretty weird and convoluted, if `calls` is an `&Option<PalletCallMetadata<_>>` you can use `.calls.as_ref()` to convert it into an `Option<&PalletCallMetadata<_>>`.

Reply:
Gotcha!